cmd/utils/flags.go

```diff
@@ -534,20 +539,6 @@ func MakeWSRpcHost(ctx *cli.Context) string {
 	return ctx.GlobalString(WSListenAddrFlag.Name)
 }
 
-// MakeGenesisBlock loads up a genesis block from an input file specified in the
-// command line, or returns the empty string if none set.
-func MakeGenesisBlock(ctx *cli.Context) string {
```

Comment:
Please see the rationale in https://github.com/ethereum/go-ethereum/pull/2788. It's removed in that PR, this one just aggregates the commits from there too.